barretenberg/cpp/src/barretenberg/goblin/translation_evaluations.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "barretenberg/ecc/curves/bn254/fq.hpp"
+
+namespace barretenberg {
+struct TranslationEvaluations {
```

Comment:
Maybe this should be in the EccOpQueue? It seems natural to have it there, since it has no goblin translator specific members.

Reply:
But then OpQueue is not a good name...